lib/services/access_checker.go

```diff
@@ -298,6 +307,32 @@ func (a *accessChecker) CheckAccess(r AccessCheckable, mfa AccessMFAParams, matc
 	return trace.Wrap(a.RoleSet.checkAccess(r, mfa, matchers...))
 }
 
+// GetKubeResources returns the allowed and denied Kubernetes Resources configured
+// for a user.
+func (a *accessChecker) GetKubeResources(cluster types.KubeCluster) ([]types.KubernetesResource, []types.KubernetesResource) {
+	var (
+		allowedResources []types.KubernetesResource
+		deniedResources  []types.KubernetesResource
+	)
```

Comment:
Use named responses? It's helpful here too.

```suggestion
func (a *accessChecker) GetKubeResources(cluster types.KubeCluster) (allowed, denied []types.KubernetesResource) {
```